src/Workspaces/Remote/Core/Serialization/RoslynJsonConverter.RoslynOnly.cs

```diff
@@ -187,5 +190,207 @@ protected override void WriteValue(JsonWriter writer, AnalyzerPerformanceInfo in
                 writer.WriteEndObject();
             }
         }
+
+        private sealed class DiagnosticDataLocationJsonConverter : BaseJsonConverter<DiagnosticDataLocation>
```

Comment:
Do we have analyzers to ensure this stays up to date with what it's serializing? Otherwise, could be a potential source generator

Reply:
Oh, this can now be removed as we moved to Message Pack. Will remove.